.travis.yml

```diff
@@ -17,7 +23,12 @@ cache:
 services:
   - docker
 script:
-  - docker-compose run test
+  - export SAUCE_HOST=$(ip -4 addr show scope global dev docker0 | grep inet | awk '{print $2}' | cut -d / -f 1)
+  - docker-compose run --service-ports test
 notifications:
   email:
-    - tech@ebmdatalab.net
\ No newline at end of file
+    - tech@ebmdatalab.net
+addons:
+  sauce_connect:
+    username: ebmdatalab
+    access_key: 397c2f14-5ab3-43cf-a47c-019807cd3c44
```

Comment:
Are we happy it's ok to have the access key here? (It may be unavoidable, but just wanted to check.)